### ISSUE TITLE
QUESTION, minimum configuration required to boot correctly?

### ISSUE THREAD

**Issue by Lukypie** [darkhandz/XPS15-9550-High-Sierra#17]

**Comment by darkhandz** [comment]
Yes, `patched`folder contains all hotpatch aml, and they are list in the config.plist->ACPI->DSDT->Patched node. 
If you want to do a minimal configuration, you can find in [Rehabman's repo](https://github.com/RehabMan/OS-X-Clover-Laptop-Config), most of my aml is copy from there 😆.

And, [wmchris's tutorial](https://github.com/wmchris/DellXPS15-9550-OSX)is in English, and more helpful.
